### PR TITLE
Add μπαμπου, βιολα as protected words

### PR DESCRIPTION
**Pull request by pharlez**

### CODE REVIEW

**Review by meletakis — APPROVED** [review]
LGTM!

**Comment by greenonion** [comment]
I think we should also update the java stemmer exceptions accordingly - though I think this can be done dynamically now? 🤔  cc @cmantas

File: config/stemmer.yml [hunk 1]
```diff
@@ -100,6 +100,7 @@ protected_words:
   - ΑΦΟΙ
   - ΑΦΟΥ
   - ΑΦΟΤΟΥ
+  - ΒΙΟΛΑ
```
Comment (pharlez) [inline config/stemmer.yml:103]: I think I should I add exceptions for other cases (e.g. `βιόλας`) and/or numbers (e.g. `βιόλες`) too?

**Comment by cmantas** [comment]
> I think we should also update the java stemmer exceptions accordingly - though I think this can be done dynamically now?  
For ElasticSearch pipelines we do indeed have the [dynamic word delimiter](https://github.com/skroutz/elasticsearch-dynamic-word-delimiter) and [dynamic stemmer override](https://github.com/skroutz/dynamic-stemmer-override).
In our organization we also have a full-fledged ruby text analysis module that mimics ES analysis pipeline (and thus also has such dynamic exceptions)
However if someone wants to directly use the greek stemmer gem, they do not have these tools.

**Review by cmantas — APPROVED** [review]

**Pull request merged**